chart/templates/ws-manager-bridge-configmap.yaml

```diff
@@ -25,6 +25,11 @@ data:
           "host": "localhost",
           "port": "8080"
         },
+        "timeouts": {
+          "metaInstanceCheckIntervalSeconds": 60,
+          "preparingPhaseSeconds": 7200,
+          "unknownPhaseSeconds": 7200
```

Comment:
should be 10min (600 seconds).
We don't expect workspaces to be in unknown for longer than a few seconds really.

Reply:
Ok, I'll change